src/model/comment.ts

```diff
@@ -28,7 +28,6 @@ export interface IDocumentComment extends Document {
 export interface IComment {
   uuid: string;
   author: Schema.Types.ObjectId;
-  articleId: Schema.Types.ObjectId;
   // If reply isn't set that meant this comment, is in the root of comment section
   reply?: Schema.Types.ObjectId;
```

Comment:
We won't and don't do multi-purpose for key. It doesn't keep data consistent.

Reply:
Many thanks for your advice. Will fix it now.